Neos.Flow/Configuration/Settings.Http.yaml

```diff
@@ -24,15 +24,21 @@ Neos:
       # request (default).
       baseUri: NULL
 
+      # Defines the PSR-15 middlewares that a request goes through in a flow
+      middlewares:
```

Comment:
This is now only a single "chain" out of the box, though it is still possible to configure chains - this could be helpful to group middlewares and sort the whole groups via positional array sorter

Reply:
I love that we remove the chain and have a "single entry point" and define them by position 😍